website/source/intro/vs/istio.html.md

```diff
@@ -41,25 +41,26 @@ updates out via Pilot. This dramatically reduces the scalability of Istio,
 whereas Consul is able to efficiently distribute updates and perform all
 work on the edge.
 
+Consul provides layer 7 features for path-based routing, traffic shifting,
+load balancing, and telemetry. Consul enforces authorization and identity to
+layer 4 only -- either the TLS connection can be established or it can't.
+We believe service identity should be tied to layer 4, whereas layer 7 should be
+used for routing, telemetry, etc. Consul will be adding more layer 7 features in the future.
+
 The data plane for Consul is pluggable. It includes a built-in proxy with
 a larger performance trade off for ease of use. But you may also use third
-party proxies such as Envoy. The ability to use the right proxy for the job
-allows flexible heterogeneous deployments where different proxies may be
-more correct for the applications they're proxying.
+party proxies such as Envoy to leverage layer 7 features. The ability to use the
+right proxy for the job allows flexible heterogeneous deployments where
+different proxies may be more correct for the applications they're proxying. We
```

Comment:
I know you didn't actually change this part, but it feels a bit like an overstatement. I just wanted to call it out in case we want to walk it back a bit while we're here.

Reply:
Hi Judith, I actually did modify this text a little bit. Could you clarify which part you found contentious? I'm happy to make any change necessary to ensure we're not overstating our capabilities here.

Reply:
Just that when I hear "pluggable" I think there will be a nice clean interface where Consul can configure the proxies really easily (like using Envoy vs the built-in proxy). But my impression is that adding some more sophisticated functionality might be different between different proxies because it would rely on the proxies themselves. 

I could totally be overestimating how complicated it is though! I'll forward the email thread that gave me that impression and maybe we can talk more about it offline 😄 In any case, I don't think this wording is a big departure from what we had before, so I don't want it to block your additions